src/backend/chat.nim

```diff
@@ -87,7 +87,7 @@ proc sendImages*(chatId: string, images: var seq[string]): RpcResponse[JsonNode]
         "imagePath": image,
         # TODO is this still needed
         # "ensName": preferredUsername,
-        "text": ""
```

Comment:
Is this a user-visible message? Shouldn't it be translatable? How about having it in the QML side as `qsTr("<...>")` based on the missing text state and integrated into the design? Or this is a quick fix and plan to follow up on it?